representation/transparency_using_selection/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { createRootViewer } from "./common/init";
+import { setStructureTransparency } from "molstar/lib/mol-plugin-state/helpers/structure-transparency";
+import { Queries, QueryContext, StructureProperties, StructureSelection } from "molstar/lib/mol-model/structure";
+
+async function init() {
+    // Create viewer
+    const plugin = await createRootViewer();
+    
+    // Download PDB
+    const fileData = await plugin.builders.data.download(
+        { url: "https://models.rcsb.org/4hhb.bcif", isBinary: true }
+    );
+
+    // Load PDB and create representation
+    const trajectory = await plugin.builders.structure.parseTrajectory(fileData, "mmcif");
+    const data = await plugin.builders.structure.hierarchy.applyPreset(trajectory, "default");
+
+    // Get Structure and StructureRef objects from data
+    const struct = data?.structure.cell?.obj?.data!;
+    const structRef = plugin.managers.structure.hierarchy.current.structures.find((sr)=>sr.cell.obj?.data === struct)
+    
+    // Query all chains with ID A or B
+    const ctx = new QueryContext(struct)
+    const query = Queries.generators.chains({
+        chainTest: l => StructureProperties.chain.label_asym_id(l.element) == 'A' || StructureProperties.chain.label_asym_id(l.element)== 'B'
+    })
+    const res = query(ctx);
```

Comment:
`result` rather than `res` which could be mistaken for a residue

Reply:
in fact `sel` or `selection` or `structureSelection` would be even better because it would read like: you get a selection by applying a query to query context based on your structure